src/IntroPage/__tests__/IntroPage.test.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import { PureIntro } from '../index';
+import { user } from '../../helpers/defaultTestProps';
+
+describe('IntroPage', () => {
+  const props = {
+    user,
+    history: {
+      push: jest.fn(),
+    },
+  };
+  it('should match snapshot', () => {
+    const wrapper = shallow(<PureIntro {...props} />);
+    expect(wrapper).toMatchSnapshot('<IntroPage />');
```

Comment:
in case of IntroPage ,it  doesn't matter much, but in general, snapshots tests doesn't make sense for big components as they tend to change too often, snapshot is long & noone is paying attention to it.

small, leaf components => shapshots are fine
bigger components => verifying the existence of particular child component with enzyme `find` or simulation of the events usually works better than huge snapshot